### PR TITLE
Improved handling of missing ssh client software

### PR DESCRIPTION
**Pull request by flenter**
also added a wercker.yml and updated the script according to shellcheck hints.

### CODE REVIEW

**Comment by flenter** [comment]
This pr should resolve #5

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Add to known_hosts
 
-A wercker step to use ssh-keyscan to add a host to the known_hosts file. This step requires 
+A wercker step to use ssh-keyscan to add a host to the known_hosts file. This step requires
```
Comment (hatchan) [inline README.md]: Looks like something is missing here

File: README.md [hunk 2]
```diff
@@ -36,6 +37,22 @@ deploy:
         fingerprint: ce:83:e9:7d:02:a4:e3:63:3f:8a:07:cc:d5:d9:bb:cd
 ```
 
+# FAQ
+
+__ Step fails with the message: "... Cause: ssh-client software probably not installed." __
```
Comment (hatchan) [inline README.md]: Other steps use `##`. This allows for deeplinking (https://github.com/wercker/step-github-upload-asset/blob/master/README.md)

**Comment by hatchan** [comment]
Small things :lgtm:

**Comment by fruitl00p** [comment]
I'd love to see this land in a new release as with the push to docker and using official images this would alleviate alot of weird errors ;)

**Comment by marshall007** [comment]
@flenter @hatchan: FYI, this broke all of our builds that needed to clone from GitHub. It appears that others in Slack are reporting the same issues, which is fixed by explicitly using `add-to-known_hosts@1.4.0`.

**Pull request merged**